gsa/src/web/pages/policies/__tests__/listpage.js

```diff
@@ -62,6 +62,12 @@ const currentSettings = jest.fn().mockReturnValue(
   }),
 );
 
+const getSetting = jest.fn().mockReturnValue(
```

Comment:
Btw. this can be simplified by using jest.fn().mockResolvedValue https://jestjs.io/docs/en/mock-function-api#mockfnmockresolvedvaluevalue